### DISCUSSION TITLE
Support error handling in Repeater mode

### DISCUSSION THREAD

**Discussion by gitdotvn** [SWRT-dev/swrt-gpl#18] [General]
Hello SWRT.
I am using ASUS GT-AC5300 Router with the latest SWRT firmware code: GTAC5300_R5.2.0_30006-g7b961c7_cferom_ubi.w.
I am using Repeater mode. When I configure the ASUS Router in this mode, I cannot connect to the internet via this Router's wifi. I can only connect to the internet using a LAN cable. I hope SWRT can help me fix this error in the next version and can increase the wifi transmit power of ASUS Router to 30 dBm.
I sincerely thank.
Best regards !

**Comment by paldier** [comment]
2G? 5G? or ALL?

**Reply by gitdotvn** [reply]
Thanks for the reply.  I confirm the error on all 2.4G and 5G bands. Please help me. Thank you very much.

**Reply by gitdotvn** [reply]
I would like to provide more information, although I can connect to the internet by LAN cable, the connection signal is really unstable, constantly disconnecting and reconnecting when in Repeater mode.

**Reply by paldier** [reply]
I have tested it, non-tcp can connect to all networks via wifi, and i also tested previous version, i think this problem is in closed source driver.

**Reply by gitdotvn** [reply]
I think so too, I tried flashing some stock firmware and found that the firmware has the code: ASUS GT-AC5300 Firmware version 3.0.0.4.386.xxxxx no Repeater function error. I noticed that the firmware has an older code than: ASUS GT-AC5300 Firmware version 3.0.0.4.384.xxxxx, the Repeater function can be used normally. Please help me. I have tried flashing all the SWRT firmwares that I have downloaded from the website but none of them can use Repeater mode.

**Comment by gitdotvn** [comment]
I did a hard reset and found out that in repeater mode, the 2.4 and 5.0 bands are all given IPs and when connecting to the wifi in the 2.4 band and the 1st band 5.0 can ping all IPs and host names like  8.8.8.8, 1.1.1.1 or www.google.com,... normally, but cannot access the internet.  I suspect it's a DNS error.  Especially when connected to the 2nd 5.0 band, I can access the internet normally.  The error of not accessing the internet only occurs on the 2.4 band and the 1st band 5.0. Please help me.

**Comment by gitdotvn** [comment]
Please help me.

**Comment by gitdotvn** [comment]
Please help me.
